### ISSUE TITLE
Write Dashboard Changes to DB

### ISSUE THREAD

**Issue by hunterjsb** [we-be/c7#3]
## Issue
We need to create a function like `c3.grade` or `c3.update` that we can call when changes are made to the dashboard, so we reflect those changes in the database. If this proves annoying to implement we can look at **using a __save__ button** instead.

## Suggested Implementation
in c3.py: function to reflect changes in Cosmos
in app.py: call function when changes are made or when ed saves

**Comment by hunterjsb** [comment]
@LePogoZ I've implemented `c3.update` in the branch 3-update-db. Here's an example of a call to the function:

```python
# only updating the grade
c3.update('e628118b-aa72-3639-adef-a459f912763e', grade='C')

# passing in all the values from a "listing_body" dict
c3.update(_id, **listing_body)

# updating two attributes on the test partition*
c3.update(e628118b-aa72-3639-adef-a459f912763e', 'test', back_dmg=True, version='iphone SE')
```

**partitioning* is an Azure Cosmos concept. Our "items" (in this case listing) in the database are partitioned based on a partition key - in our case it's the 'status' of the listing. It's just a string key. Status can be "Active", "Test", "Pending", "Stale", and so on. See c3.py for the implementation. See https://learn.microsoft.com/en-us/azure/cosmos-db/partitioning-overview for info on partitioning and partition keys.

**Comment by hunterjsb** [comment]
fixed with #8
